source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/direct/cartpole/cartpole_camera_env.py

```diff
@@ -52,9 +47,13 @@ class CartpoleRGBCameraEnvCfg(DirectRLEnvCfg):
         width=80,
         height=80,
     )
-    num_observations = num_channels * tiled_camera.height * tiled_camera.width
     write_image_to_file = False
 
+    # spaces
+    action_space = 1
+    state_space = 0
+    observation_space = [tiled_camera.height, tiled_camera.width, 3]
```

Comment:
just curious, can this still be specified through hydra?

Reply:
For `Isaac-Cartpole-RGB-Camera-Direct-v0` the default observation is `Box(-inf, inf, (1024, 80, 80, 3), float32)`

After defining the CLI argument `env.observation_space=[80,128,1]` (as changed in docs [here](https://github.com/isaac-sim/IsaacLab/pull/1117/files#diff-3a1b245e379bb3ecfe0d8d542dfa467c7740f560c91e7047ece16e0f3cae8f65)) and print the observation scape it will be `Box(-inf, inf, (1024, 80, 128, 1), float32)` 

Note that including spaces (e.g.: `env.observation_space="[80, 128, 1]"` or `env.observation_space=[80, 128, 1]`) doesn't work:

```
[INFO]: Parsing configuration from: /home/toni/Documents/RL/isaac-sim_IsaacLab/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/direct/cartpole/agents/rl_games_camera_ppo_cfg.yaml
no viable alternative at input '[80,'
See https://hydra.cc/docs/1.2/advanced/override_grammar/basic for details
```